contributors/triage/guideline.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes Issue Triage Guideline
+
+## Purpose
+
+Speed up issue management.
+
+The Kubernetes issues are listed at https://github.com/kubernetes/kubernetes/issues. The issues are identified with labels. For example, an issue that belongs to SIG Network will eventually be set to label `sig/network`. New issues will start out without any labels. The detailed list of labels can be found at https://github.com/kubernetes/kubernetes/labels. While working on triaging issues you may not have privilege to assign specific label (e.g. `triaged`) and in that case simply add a comment in the issue with your findings.
+
+## Scope
+
+This guideline serves as a primary document for triaging incoming issues to Kubernetes. SIGs and projects are encouraged to either use this guideline, or use this as a starting point if necessary. For example if your SIG has specific triaging needs, extend this guideline. 
+
+## Using the bot
+
+Most people do not have the ability to interact with the github issues and labels, for that we use a bot to manage labelling and triaging. The bot has a set of [commands and permissions](https://github.com/kubernetes/test-infra/blob/master/commands.md), this document will cover the basic ones. 
+
+
+## Determine if it’s an issue or support request
```

Comment:
By issue do we mean bug, feature request, etc. ?

The term "issue" might be confusing since it's always represented as a Github issue.